component-samples/scripts/keys_gen.sh

```diff
@@ -0,0 +1,247 @@
+#!/bin/bash
+#
+# Copyright 2021 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+
```

Comment:
Add comments about what this script does and also explains the naming of different files.

That way, we don't have to add metadata information in the password / hash files.

Reply:
Added all the comments.